### DISCUSSION TITLE
[v4] flex grow

### DISCUSSION THREAD

**Discussion by cuquo** [tailwindlabs/tailwindcss#15068] [Help] [answered]
testing the latest alpha and I noticed that `flex-grow` was no longer working. It seems it was changed to `grow`

**Comment by wongjn** [comment]
Could be a breaking change in v4. [You should be using `grow` anyway since v3](https://tailwindcss.com/docs/upgrade-guide#flex-grow-shrink).

#### Edit

Yep, beta docs just came out and[ `flex-grow-*` is removed in favour of `grow-*`](https://tailwindcss.com/docs/v4-beta#removed-deprecated-utilities):

> We’ve removed any utilities that were deprecated in v3 and have been undocumented for several years. Here’s a list of what’s been removed along with the modern alternative:
>
> | Deprecated | Replacement |
> | -- | -- |
> | `flex-grow-*` | `grow-*` |

**Reply by adamwathan** [reply, marked as answer]
Brought it back last minute actually too 🙈 So should work in the beta!

https://github.com/tailwindlabs/tailwindcss/pull/15069
